rules/exist/exist.js

```diff
@@ -0,0 +1,11 @@
+const glob = require("glob");
+
+module.exports = function exist({glob: globPattern}) {
+  files = glob.sync(globPattern);
```

Comment:
It actually occurred to me earlier that we prob want the rules to be async? We can chat thru that tomorrow and see what makes sense

Reply:
ok let's def talk about that